configs/pafpn/faster_rcnn_r50_fpn_1x_coco.py

```diff
@@ -0,0 +1,12 @@
+_base_ = [
+    '../_base_/models/faster_rcnn_r50_fpn.py',
+    '../_base_/datasets/coco_detection.py',
+    '../_base_/schedules/schedule_1x.py', '../_base_/default_runtime.py'
+]
```

Comment:
We may inherit from `../faster_rcnn/faster_rcnn_r50_fpn_1x_coco.py`. 
Also, rename this file to `faster_rcnn_r50_pafpn_1x_coco.py`.